oidc_apis/backchannel_logout.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+import requests
+from oidc_provider.lib.utils.common import get_issuer
+from requests import RequestException
+
+from oidc_apis.models import ApiScope
+from tunnistamo.oidc import create_logout_token
+
+logger = logging.getLogger(__name__)
+
+
+def send_backchannel_logout_to_api(api, request, sub, sid=None):
+    if not api.backchannel_logout_url:
+        return
+
+    iss = get_issuer(request=request)
+
+    response = requests.post(api.backchannel_logout_url, data={
+        'logout_token': create_logout_token(api.oidc_client, iss, sub, sid),
+    })
+    response.raise_for_status()
+
+
+def send_backchannel_logout_to_apis_in_token_scope(oidc_token, request, sid=None):
+    known_api_scopes = ApiScope.objects.by_identifiers(oidc_token.scope)
+    token_api_scopes = known_api_scopes.allowed_for_client(oidc_token.client)
+
+    for api_scope in token_api_scopes:
```

Comment:
There might be network timeouts or such things which might make this loop operation get stuck for a long while (visible to the user?). Also, in case of errors, do we ever want to retry? Would a job queue style more async implementation be more quick/robust?

Reply:
Yeah, maybe should add a short timeout value. Asynchronous requests would definitely be the correct choice. But we don't have Celery or anything like that in use in Tunnistamo.

Reply:
A short timeout value ok for now.